### PR TITLE
Download Edge version via Docker Hub

### PR DESCRIPTION
**Pull request by StefanScherer**
Signed-off-by: Stefan Scherer <stefan.scherer@docker.com>

### Proposed changes

Update the link to download the latest Docker Desktop Edge version via Docker Hub. The previous link downloads 2.1.6.0 and we want users to get the current Edge version.

### CODE REVIEW

File: docker-for-windows/wsl-tech-preview.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ Before you install Docker Desktop WSL 2 backend, you must complete the following
 
 # Download
 
-Download [Docker Desktop Edge 2.1.6.0](https://download.docker.com/win/edge/40807/Docker%20Desktop%20Installer.exe) or a later release.
+Download [Docker Desktop Edge 2.1.6.0](https://hub.docker.com/editions/community/docker-ce-desktop-windows/) or a later release.
```
Comment (usha-mandya) [inline docker-for-windows/wsl-tech-preview.md]: Thanks Stefan. Can we move the version number (2.1.6.0) out of the link?

Comment (StefanScherer) [reply docker-for-windows/wsl-tech-preview.md]: Good catch. I've updated the PR.

**Comment by thaJeztah** [comment]
The page on Docker Hub on its hand points back to `download.docker.com` (https://download.docker.com/win/edge/Docker%20Desktop%20Installer.exe) wondering if we need the "Docker Hub" link in between, or could just point to that URL?

**Comment by usha-mandya** [comment]
@thaJeztah Thanks for chiming in. We discussed this with @nebuk89 and agreed to point users to https://hub.docker.com/editions/community/docker-ce-desktop-windows for download URLs.

**Pull request merged**